synced_versions_formulae.json

```diff
@@ -32,6 +32,7 @@
   ["libnetworkit", "networkit"],
   ["libnghttp2", "nghttp2"],
   ["libngspice", "ngspice"],
+  ["libvirt", "libvirt-python"],
```

Comment:
if it autobumped, then you cannot do sync

Reply:
why does it need sync?

Reply:
Because they are always being released simultaneously. See:

https://lists.libvirt.org/archives/list/announce@lists.libvirt.org/
https://download.libvirt.org/python/
https://download.libvirt.org/

Reply:
But it's not technically required for these packages to be in sync, consider this as just a hint for maintainers to bump them in one PR.

Reply:
currently, autobump does not support bumping two formulae at the same time, if they dont have inter-dep between the two, I would suggest remove them from sync (they would be bumped at the same time and kept up to date). let me know if that makes better sense.

Reply:
I see, that's makes sense, they'll better be bumped in two separate PRs at the same `autobump` run. So I dropped the sync commit.